scripts/yaml_characters

```diff
@@ -22,8 +24,16 @@ OptionParser.new do |opts|
     excluded_locales << locale.to_sym
   end
 
+  opts.on('--exclude-path=PATH', 'Disregard characters from the given relative path') do |path|
+    excluded_paths << File.join(Dir.pwd, path, '')
+  end
+
   opts.on('--exclude-gem-path=GEM', 'Disregard characters loaded by the given gem') do |gem|
-    excluded_gem_paths << Gem.loaded_specs[gem].full_gem_path
+    excluded_paths << File.join(Gem.loaded_specs[gem].full_gem_path, '')
```

Comment:
The reason for the `File.join` ending with `''` (as with above) is to normalize a trailing slash.

Without, it could be possible for the `.start_with?` check below to match `config/locales/telephony_not_the_one_i_mean/en.yml` if given `--exclude-path=config/locales/telephony` .